### PR TITLE
boards: add new Holybro Pix32v5 board

### PR DESCRIPTION
**Pull request by dagar**
TODO:

1. Connection for TELEM3,
2. Added a heating circuit for the IMU.

### CODE REVIEW

**Comment by vincentpoont2** [comment]
Schematic Diagrams for the added TELEM3 and heating element.

TELEM3 Circuit
Heating circuit

**Comment by vincentpoont2** [comment]
@jamming

File: boards/holybro/pix32v5/firmware.prototype [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "board_id": 50,
```
Comment (davids5) [inline boards/holybro/pix32v5/firmware.prototype:2]:
```suggestion
    "board_id": 78,
```

**Comment by vincentpoont2** [comment]
@dagar Will you be able to help make these changes?

- TELEM3 & heater changes on default.cmake file?
- Heater code on rc.board_sensors?
- manifest.c file changes for new board?
- others?

**Comment by mrpollo** [comment]
@vincentpoont2 Can you please get @dagar hardware for the test rack?

**Comment by vincentpoont2** [comment]
> @vincentpoont2 Can you please get @dagar hardware for the test rack?

Definitely, do you have an address or should I dm Daniel?

**Comment by jinger26** [comment]
@vincentpoont2  DM @dagar for address

File: boards/holybro/pix32v5/firmware.prototype (hunk 1 above)
Comment (dagar) [reply boards/holybro/pix32v5/firmware.prototype]: Done.

**Comment by dagar** [comment]
Telem 3 added. I'll leave the heater alone until I have hardware in hand.

**Comment by vincentpoont2** [comment]
@dagar Can Holybro send you the hardware now? Can you DM your address? Thanks!

**Pull request merged**